tests/test.sh

```diff
@@ -1051,6 +1051,10 @@ if test -x $CLANGXX; then
     rm "$testdir"/plain.o
     run_ice "" "remote" 0 $CLANGXX -Wall -Werror -c includes.cpp -o "$testdir"/includes.o
     rm "$testdir"/includes.o
+    run_ice "" "remote" 0 $CLANGXX -Wall -Werror -cxx-isystem ./ -c includes.cpp -o "$testdir"/includes.o
+    rm "$testdir"/includes.o
+    run_ice "" "remote" 0 $CLANGXX -Wall -Werror -target x86_64-linux-gnu -c includes.cpp -o "$testdir"/includes.o
```

Comment:
will this potentially fail on non x86 platforms if the correct headers are not installed?